SD_DSC.adoc

```diff
@@ -1485,15 +1485,15 @@ The evaluator shall also examine the TSS to determine the behaviour that will oc
 
 ====== AGD
 
-The evaluator shall examine the guidance documentation to ensure that instructions for configuring the number of successive unsuccessful authentication attempts and time period (if implemented) are provided, and that the process of unlocking the SDOs is described for each "action" specified (if that option is chosen). 
+If the administrator is able to configure any of the variables for authorization attempts, the evaluator shall examine the guidance documentation to ensure that instructions for configuring the number of successive unsuccessful authentication attempts and time period (if implemented) are provided, and that the process of unlocking the SDOs is described for each "action" specified (if that option is chosen). 
```

Comment:
FMT_SMF.1 actually requires that the TSF is capable of performing the "Set authorization failure parameters for FIA_AFL_EXT." management function. If we want this to be optional, I believe it should be changed there.

Because this is defined in FMT_SMF.1, I feel like the first part of this sentence ("instructions for configuring the number of successive unsuccessful authentication attempts and time period (if implemented) are provided") should be evaluated as part of the AGD for FMT_SMF.1.

Reply:
I concur.

Reply:
So we can change FMT_SMF.1, but the SFR is written to make this optional. FIA_AFL_EXT.1.2 states that the options are "static, administrator configurable variable" so by definition we are saying that in some cases the system may come configured without an option to change it. I actually think this is what is most likely for most DSC implementations anyway, but this is part of the SFR and has been, I didn't change that. The change here was because of what is in the SFR since you can't test a configurable setting if there isn't one.